### PR TITLE
Add search command

### LINKED ISSUE #177: Add !jira search command
It would be nice to have a !jira search command, similar to the !wiki search command on the Wiki-Bot

### PR DESCRIPTION
**Pull request by chandler05**
## Purpose
Close #177

### CODE REVIEW

**Comment by dericksonmark** [comment]
Overall, this looks pretty good! There seem to be some minor formatting/escaping issues, which I'm not sure are worth the time to fix.

- When using `!jira search <url>` ( with the `<>`), Discord thinks that MojiraBot pasted two links, and separates them accordingly. This also leaves an extra `%22` at the end of the link.
- When using `!jira search "text"` (with the `""`), the quotes in the query template are being closed, leading to an invalid query and returning no results.

**Comment by chandler05** [comment]
The first one is not worth fixing, but I'll look into the second

**Comment by chandler05** [comment]
Managed to fix both anyways

**Comment by dericksonmark** [comment]
It might also be nice to allow for the deletion of the search result embeds.

**Comment by chandler05** [comment]
Added

**Review by violine1101 — APPROVED** [review]
Simple and straightforward. Neat!
Obviously there's room for improvement, but for a first iteration of a command this is totally fine.
For instance, it'd be neat to have the JQL configurable in the config too.
I'd merge this as is now, but what I've mentioned should probably be done in a followup PR.

File: src/commands/SearchCommand.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	public async run( message: Message, args: string ): Promise<boolean> {
+		if ( !args.length ) {
+			return false;
+		}
+
+		const plainArgs = args.replace( /"|<|>/g, '' );
+
+		try {
+			const embed = new MessageEmbed();
+			const searchFilter = `text ~ "${ plainArgs }"`;
```
[the remark this answers is not in the document]
Comment (chandler05) [reply src/commands/SearchCommand.ts:19]: I'll update it to read from the projects in the config

**Pull request merged**